WordPress/Docs/Classes/ClassInstantiationStandard.xml

```diff
@@ -0,0 +1,53 @@
+<documentation title="Class Instantiation">
+    <standard>
+    <![CDATA[
+    Instantiation of an object should be done with parenthesis.  
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: with parenthesis.">
+        <![CDATA[
+$a = new Foobar<em>()</em>;
+        ]]>
+        </code>
+        <code title="Invalid: without parenthesis.">
+        <![CDATA[
+$a = new Foobar;
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Don't use spaces between object name and open parenthesis when instantiating new object.  
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: no spaces between object name and parenthesis.">
+        <![CDATA[
+$a = new Foobar();
+        ]]>
+        </code>
+        <code title="Invalid: spaces between object name and parenthesis.">
+        <![CDATA[
+$a = new Foobar<em> </em>();
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Object instantiation by reference is not supported by PHP anymore.  
```

Comment:
I know I wasn't too clear about what this error is about on Friday, so I've looked up a better/more accurate description now.

```suggestion
    Assigning the return value of "new" by reference was deprecated in PHP 5.3 and removed in PHP 7.0. New by reference should no longer be used.
```